scripts/setup_imdb.py

```diff
@@ -16,9 +17,14 @@ def clean_up(including_table_dir=False):
         os.remove(FILE_NAME)
 
     if including_table_dir and os.path.exists(table_dir):
-        for file in os.listdir(table_dir):
-            os.remove("./%s/%s" % (table_dir, file))
-        os.rmdir(table_dir)
+        try:
+            shutil.rmtree(table_dir)
+        except PermissionError as e:
+            print(f"PermissionError: Unable to remove {table_dir}. Please check file permissions.")
+            raise
+        except Exception as e:
+            print(f"An error occurred while cleaning up {table_dir}: {e}")
+            raise
```

Comment:
While I like condensing the deletion of the directory into a single call, I am not sure about the value added by catching. I think the errors are telling enough (esp. `PermissionError`). In the other places, we only catch to cleanup (so hopefully re-trying downloads the data again and that already helps), but IMO we can simply crash if the cleanup itself does not work.

Reply:
Agreed.